app/helpers/feature_flag.rb

```diff
@@ -1,7 +1,15 @@
 module FeatureFlag
   extend self # rubocop:disable Style/ModuleFunction
```

Comment:
I forgot to comment on this in the original PR: we currently do this in several different ways across the code-base:

1. `extend self`
2. Prefixing every method name with `self.`
3. `class << self`
4. `module_function`

For the sake of consistency, we should probably settle on one and given that the one used here needs a temporary Rubocop override I'd vote for 2. or 3. (unless you need the extra feature of `module_function`, in which case 4. is ok).

Reply:
Good point: I pretty much took this from the definition of `Flipper::DSL`. I'll opt for option 2 :)